prac_09/band.py

```diff
@@ -0,0 +1,34 @@
+"""Band class for CP1404"""
+
+
+class Band:
+    """Band class"""
+
+    def __init__(self, name=""):
+        """Construct a Band with a name and empty list of members."""
+        self.name = name
+        self.members = []
+
+    def __str__(self):
+        """Return a string representation of a Band."""
+        member_strings = []
```

Comment:
This could be shortened using list comprehensions e.g. `[str(member) for member in self.members]`

Reply:
Good point! Thanks for spotting that! 🐾